package.json

```diff
@@ -8,15 +8,15 @@
     "build": "babel --presets es2015 --plugin add-module-exports src --out-dir lib",
     "build:umd": "webpack --config ./webpack/webpack.config src/index.js dist/nordnet-next-api.js && NODE_ENV=production webpack --config ./webpack/webpack.config src/index.js dist/nordnet-next-api.min.js",
     "eslint": "eslint src || echo \"Linting failed...\"",
-    "lint": "npm run eslint",
+    "lint": "run-s eslint",
     "pretest": "rimraf reports/test-results.xml reports/coverage",
     "test": "karma start ./test/karma.conf.js --reporters mocha,junit,coverage --source-map",
     "test:debug": "karma start ./test/karma.conf.js --no-single-run --reporters mocha --browsers Chrome --source-map",
     "test:tdd": "karma start ./test/karma.conf.js --auto-watch --no-single-run ",
-    "validate": "npm run lint && npm test",
-    "prebuild": "npm run validate",
-    "prepush": "npm run validate",
-    "prepublish": "npm run build",
+    "validate": "run-s lint test",
+    "prebuild": "run-s validate",
+    "prepush": "run-s validate",
+    "prepublish": "run-s build",
```

Comment:
do even need `npm-run-all` after this refactoring? i mean primary use case is go run several npm scripts either sequentially or in parallel. in our case we run single scripts

Reply:
We might not. I assumed it was needed to be yarn/npm agnostic.

Reply:
> If a script is invoked with Yarn, npm-run-all will correctly use Yarn to execute the plan's child scripts.

Reply:
And what if it's invoked with npm?

Reply:
scripts in turn will be invoked with npm